spec/models/sms/sms_decoder_spec.rb

```diff
@@ -6,25 +6,25 @@
 
   describe "authentication" do
     context "with SMS Authentication enabled" do
-      before(:all) { create_form(questions: %w(integer), authenticate_sms: true) }
+      let(:form) { create_form(questions: %w(integer), authenticate_sms: true) }
```

Comment:
Thanks for making these changes, fingers crossed that we stop getting these spec errors on the SMS decoding! 🤞